communities/hacs/readme.md

```diff
@@ -0,0 +1,38 @@
+[![Build Status](https://travis-ci.org/HaCSBCU/hacsbcu.github.io.svg?branch=master)](https://travis-ci.org/HaCSBCU/hacsbcu.github.io)
+
+![Website Screenshot](http://i.imgur.com/lJnqm1B.jpg)
```

Comment:
Just being picky really but I noticed two things which might be ever so minor improvements.

1. With Chrome if you hover in between the address bar and your addons, you get the option to drag the size of your addons menu around. When I'm doing screenshots I tend to crush this down to nothing, it's not nessesary but if you're looking for an extra clean finish it is a quick thing to do.
2. The image you're linking to is on imgur, this is just a personal preference thing but I tend to keep even these images within the repo as they're part of your content.

Again, it's super picky and not anything which will greatly improve the readme (because it's already top notch) but maybe something to consider.